src/components/button/index.js

```diff
@@ -2,31 +2,34 @@ import React, { Component } from 'react'
 import './index.less'
 import classNames from 'classnames';
 
-export default class Button extends Component {
+export default class TSButton extends Component {
     constructor() {
         super()
     }
 
     render() {
+        var cn = this.props.className
         const buttonStyle = {
             color: `${this.props.color}`,
             height: `${this.props.height}px`,
             borderRadius: `${this.props.borderRadius}px`,
+            width: `${this.props.width}px`
 
         };
         var btnClass = classNames({
             'bt-primary': this.props.type === 'primary',
             'bt-outline': this.props.type === 'default',
-        });
+        }, this.props.className);
         return <button className={btnClass} style={buttonStyle}> {this.props.children}</button>
     }
 }
 
```

Comment:
In React, every component that have props passed in should have propTypes. Reference this link: https://reactjs.org/docs/typechecking-with-proptypes.html